.rubocop.yml

```diff
@@ -3,4 +3,7 @@ AllCops:
     - vendor/**/*
     - bin/**/*
 
+Style/DoubleNegation:
```

Comment:
Na, just remove it from here and let the auto-gen-config flag it as a violation. Maybe one day we'll change our mind on whether we like `!!` or not.